tests/console/rmt.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add rmt configuration test
+#    test installation and upgrade with rmt pattern, basic configuration via
+#    rmt-wizard and validation with rmt-cli repos rmt-cli scc sync return value
+# Maintainer: Jiawei Sun <jwsun@suse.com>
+
+#use base "x11test";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit